macros/SpecName.ejs

```diff
@@ -495,7 +495,7 @@ var specList = {
     },
     'DOM3 Events':{
         name : 'Document Object Model (DOM) Level 3 Events Specification',
-        url  : 'https://w3c.github.io/uievents/'
+        url  : 'https://www.w3.org/TR/2009/WD-DOM-Level-3-Events-20090908/'
```

Comment:
there's a newer version at https://www.w3.org/TR/2014/WD-DOM-Level-3-Events-20140925/ (the last public WD named as DOM3 Events). is thre any reason pointing to the 2009 snapshot?

Reply:
I thought they all redirected to UI Events. I was trying to eliminate confusion. Fix it now.